lib/insights/api/common/application_controller_mixins/exception_handling.rb

```diff
@@ -7,19 +7,25 @@ module ExceptionHandling
 
           def self.included(other)
             other.rescue_from(StandardError, RuntimeError) do |exception|
-              logger.error("#{exception.class.name}: #{exception.message}\n#{exception.backtrace.join("\n")}")
-              errors = Insights::API::Common::ErrorDocument.new.tap do |error_document|
-                exception_list_from(exception).each do |exc|
-                  if api_client_exception?(exc)
-                    api_client_errors(exc, error_document)
-                  else
-                    error_document.add(error_code_from_class(exc).to_s, "#{exc.class}: #{exc.message}")
-                  end
-                end
+              rescue_from_handler(exception) do |error_document, exc|
+                error_document.add(error_code_from_class(exc).to_s, "#{exc.class}: #{exc.message}")
```

Comment:
@eclarizio is the exc.class too much information to the user? Should we just have the message.
Also now that we have the JSON-API errors array, the HTTP status code might get handled by the UI and it might display Authorization error for the 403 and the errors array will have the details.

https://projects.engineering.redhat.com/browse/RHCLOUD-5966

Reply:
@mkanoor Possibly `exc.class` is too much information for the user, but this is just a refactor, I didn't change any logic. I think if we want to change the functionality here, that should be a separate PR. This is simply to extract the handling code into a non-anonymous method so that it is usable.